docs/setup/production.asciidoc

```diff
@@ -125,4 +126,17 @@ elasticsearch.hosts:
 --------
 
 Related configurations include `elasticsearch.sniffInterval`, `elasticsearch.sniffOnStart`, and `elasticsearch.sniffOnConnectionFault`.
-These can be used to automatically update the list of hosts as a cluster is resized.  Parameters can be found on the {kibana-ref}/settings.html[settings page].
\ No newline at end of file
+These can be used to automatically update the list of hosts as a cluster is resized.  Parameters can be found on the {kibana-ref}/settings.html[settings page].
+
+[float]
+[[memory]]
+=== Memory
+Kibana has a default memory limit of 1.4 GB, and in most cases, we recommend leaving this unconfigured.  In some scenarios, such as running on containers with limited
```

Comment:
While the max old space is generally 1.4GB, it is based on system information, like memory available. @mistic did a deep dive into this a while back and can comment.

Reply:
ah right, good catch

Reply:
@tylersmalley exactly. Nodejs calculates the default allocated heap according multiple variables, processor architecture, total available physical memory etc (the logic can be found here https://github.com/v8/v8/blob/master/src/heap/heap.cc#L227)

So in order to be able to specify what is our minimum required memory we need to proceed what we have been discussing of distribute kibana with an override NODE_OPTIONS for heap. So, at least for now, we should not specify that information on the docs

Reply:
I added a clarifier,  `default _maximum_ memory limit.  it's not very specific but I'm hoping it's good enough